SConstruct

```diff
@@ -3422,6 +3422,21 @@ if doConfigure :
 		appleseedTestEnv.Alias( "testAppleseed", appleseedTest )
 
 ###########################################################################################
+# Install Cortex 10 forward-compatibility for IECoreImage
+###########################################################################################
+
+coreImagePythonModuleEnv = pythonModuleEnv.Clone( IECORE_NAME = "IECoreImage" )
+
+if doConfigure :
```

Comment:
Do we need the `if doConfigure` here? I'm a little hazy on the details, but I would expect that because we're not doing and configuration in the block below, we can omit the conditional. I wonder if instead it should be `if env["WITH_CORTEX10_COMPAT"], so you have to request the installation specifically by the command line args?